src/runtime/HalideBuffer.h

```diff
@@ -474,7 +474,6 @@ class Buffer {
     }
 
     Buffer() {
-        memset(&buf, 0, sizeof(halide_buffer_t));
```

Comment:
Because the default member initializer does it and we weren't doing it in other constructors so the code confused me.